[Ruleset Engine, BOTS] Update documentation with new Bot Detection IDs field (#7474)
cloudflare/cloudflare-docs: content/bots/reference/bot-management-variables.md

- **Serves Static Resource**: An identifier that matches [file extensions](/bots/reference/static-resources/) for many types of static resources. Use this variable if you send emails that retrieve static images.
- **ja3Hash**: A [**JA3 Fingerprint**](/bots/concepts/ja3-fingerprint/) helps you profile specific SSL/TLS clients across different destination IPs, Ports, and X509 certificates.
- **js_score**: Customers should not use `js_score` when creating Bot Management firewall rules because it will always be blank.

> - **Bot DetectionIDs**: List of IDs that correlate to the Bot Management Heuristic detections made on a request (Multiple Heuristics can be made on a single request)

Reviewer: ```suggestion
- **Bot DetectionIDs**: List of IDs that correlate to the Bot Management heuristic detections made on a request (you can have multiple heuristic detections on the same request).
```

These variables are also available as part of the [request.cf](/workers/runtime-apis/request/#incomingrequestcfproperties) object via Cloudflare Workers:

- `request.cf.botManagement.score`
- `request.cf.botManagement.verifiedBot`
- `request.cf.botManagement.staticResource`
- `request.cf.botManagement.ja3Hash`
- `request.cf.botManagement.js_score`

> - `request.cf.botManagement.detectionIds`

Reviewer 2: Is it worth documenting them at https://developers.cloudflare.com/workers/runtime-apis/request/#incomingrequestcfproperties too? The existing properties are available there (albeit missing `js_score too).

<img width="623" alt="image" src="https://user-images.githubusercontent.com/94662631/216646091-bc7a1567-59e3-4810-8097-7e9f5cf13a64.png">

Reviewer 2: Sounds good to me - I don't see it anyhow.

```sh
➜  ~ curl https://kian.org.uk/cf.json | jq '.botManagement'
{
  "corporateProxy": false,
  "verifiedBot": false,
  "staticResource": false,
  "detectionIds": [
    33554817
  ],
  "score": 1
}
```
